ts/session/sending/MessageQueue.ts

```diff
@@ -1,54 +1,167 @@
+import * as _ from 'lodash';
+import * as Data from '../../../js/modules/data';
```

Comment:
You'll fail to stub any data methods this way.. We had the same problem in the cache remember.